### PR TITLE
Strip symbols of all binaries

### PR DESCRIPTION
**Pull request by abrown**
@sbc100, this resolves the discussion we had in #108 about stripping symbols from all binaries.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -164,7 +164,7 @@ build/config.BUILT:
 build: build/llvm.BUILT build/wasi-libc.BUILT build/compiler-rt.BUILT build/libcxxabi.BUILT build/libcxx.BUILT build/config.BUILT
 
 strip: build/llvm.BUILT
-	cd $(PREFIX)/bin; strip clang-* clang++-* lld llvm-ar
+	./strip_symbols.sh $(PREFIX)/bin
```
Comment (sbc100) [inline Makefile]: Given the existence of this script I wonder if we should just pass `$(PREFIX)`?   Maybe there are ELF files in other directories and we want to strip those too, no?

Comment (abrown) [reply Makefile]: I'm not opposed to doing that; I just was concerned about stripping symbols where I shouldn't. Are we sure it's ok to do it on the entire `$(PREFIX)`?

Comment (sbc100) [reply Makefile]: This is the approach the debian packages normally take IIRC.. strip *all* ELF files by default.  I see no reason not to do it here... but I could be missing something.

**Comment by sunfishcode** [comment]
Testing for "ELF" is too narrow since we also support on-ELF hosts such as macOS. Would it work to just unconditionally run strip on every file under `bin` and ignore errors?

Alternatively, if we could add "-s" to the link command-line, that would cause it to strip executables too.

**Comment by abrown** [comment]
@sunfishcode, I didn't really know all the places `-s` might be needed so I tried the "strip everything and ignore errors" approach. Note that due to the discussion above with @sbc100 (resolved now) we are stripping everything, not just the `bin` directory.

**Comment by sbc100** [comment]
I think post-stripping is probably best. 

The current approach of this patch is to strip anything that `find` things is executable.  Seems reasonable to me that we just stick with that approach. Dan?

**Comment by sunfishcode** [comment]
Sounds reasonable.

**Pull request merged**